documentation/modules/ROOT/pages/creatingPatterns.adoc

```diff
@@ -82,8 +82,165 @@ We want to maximize modularity in the framework. This will reduce our reliance o
+
+
+== Introducing new operators
+
+Operators can be simply added to the clusterGroup as subscriptions
+
+[source,yaml]
+====
+  subscriptions:
+    acm:
+      name: advanced-cluster-management
+      namespace: open-cluster-management
+      channel: release-2.9
+    s3:
+      name: ack-s3-controller
+      namespace: ack-system
+      channel: alpha
+      source: community-operators
+    cte:
+      name: cte-k8s-operator
+      namespace: openshift-operators
+      channel: stable
+      source: certified-operators
+====
+
+To find the *correct* names the easiest way to do this is to jump into the console, find the operator, get to the install screen
+
+image::operator-install.png[]
+
+The URL at this point can be used to pull out the name (pkg), catalog, channel etc.
+
+
+== Introducing new helm charts
+
+The easiest way to start with new helm charts is to copy and paste from the example charts in multicloud-gitops.
+
+
+
+
+== Chaining configuration into the charts.
+
+One key element of validated patterns is a structured set of variable overrides. 
+These overrides are key to allowing the template to be kept as 'DRY' as possible when configuring.
+For this context it's important to understand the evaluation order:
+
+1. The default values defined in the chart (or sub charts).
+2. `values-global.yaml` file
+3. Values within clusterGroup files (e.g. `values-hub.yaml)
+4. Values override files with precedence for the (top/bottom) of the valid list
+5. Values overrides shown in individual application definitions
+
+
+
+*Default files installed in a chart*
+
+All helm charts *MUST* contain a `values.yaml` file.
+Values that are defined here should be considered the 'base' consideration.
+While it's possible to not define a value here, yet use it in the chart, it is strongly recommended sensible defaults are included.
+
+
+*`values-global.yaml` file*
+
+Values here are applied to every argocd application as it is picked up directly by the validated patterns operator.
+The values here are also usable in the `clusterGroup` files
+
+
+*ClusterGroup* files overrides
+
+Values can directly be used within the context of files such as `values-hub.yaml` 
+
+[source,yaml]
+====
+  letsencrypt:
+    region: ''
+    server: https://acme-v02.api.letsencrypt.org/directory
+    # staging URL
+    # server: https://acme-staging-v02.api.letsencrypt.org/directory
+    email: chris@thebutlers.me
+  clusterGroup:
+    name: hub
+    isHubCluster: true
+
+====
+
+
+*Values override files*
+
+`clusterGroup`s define an override file list which is evaluated in order.
+The cluster overrides files can take of the variables defined in `values-global.yaml` as well as metadata assembled by the operator, OpenShift and {rhacm}.
+
+These dynamic variables allow abstraction from variability between OpenShift installation. `storageClasses` varying across `clusterPlatform` is a clear example.
```

Comment:
Maybe let's add a note here that if you use variables in the overrides ($.Values...) they will be resolved by ACM *on* the hub and not on the spoke (it only affects multicluster scenarios with differing platforms/cluster versions, but it is worth noting)